postgres/src/test/java/org/jdbi/v3/pg/TestSqlArrays.java

```diff
@@ -87,7 +98,7 @@ public void testIntList() throws Exception {
         UUID[] fetchUuidArray();
 
         @SqlUpdate(U_INSERT)
-        void insertUuidArray(UUID[] u);
+        void insertUuidArray(@Bind("uuids") UUID[] u);
```

Comment:
Yes. This is the default setting when you run the Maven build. In your IDE you need to enable the `-parameters` compiler option.